tools/src/S3.ts

```diff
@@ -198,7 +195,7 @@ export async function uploadDirectoriesAsync(
         }
       );
       let gitCommandOutput = gitCommand.stdout.toString();
-      await fsWriteFileAsync(excludeFile, gitCommandOutput);
+      await fse.writeFile(excludeFile, gitCommandOutput);
```

Comment:
Reason: promisify seems redundant for fse, also is not applying right types to the wrapped function